src/tracks/InstrumentTrack.cpp

```diff
@@ -124,6 +124,7 @@ InstrumentTrack::InstrumentTrack( TrackContainer* tc ) :
 	connect( &m_baseNoteModel, SIGNAL( dataChanged() ), this, SLOT( updateBaseNote() ) );
 	connect( &m_pitchModel, SIGNAL( dataChanged() ), this, SLOT( updatePitch() ) );
 	connect( &m_pitchRangeModel, SIGNAL( dataChanged() ), this, SLOT( updatePitchRange() ) );
+	connect( &m_mutedModel, SIGNAL( dataChanged() ), this, SLOT( muteHasChanged() ) );
```

Comment:
Hmm... This seems to be the only slot which changes the GUI.  I'm not very good at this stuff but is there a chance this should be a `InstrumentTrackView` slot instead which is called with a `Qt::QueuedConnection` ?